app/views/devise/registrations/start.html.slim

```diff
@@ -1,11 +1,12 @@
 - title t('titles.registrations.start')
 
-h1.h3.my0 = t('headings.create_account')
+h1.h3.my0 = t("headings.create_account_#{service_provider}", sp: @sp_name)
```

Comment:
I agree with @zachmargolis that a conditional like he showed is more obvious than interpolation. However, this *is* a pattern we are going to be seeing more and more from the content side, per conversations with @esgoodman. "A pattern" in this context meaning that there is service provider-specific text and then an alternative for when a service provider is not present.

Given that understanding, along with @monfresh's point about "making code open to the future" I do think we are going to end up with some type of polymorphism. 

I agree that interpolating into I18n strings can be confusing / opaque. Perhaps a way to avoid that while embracing polymorphism would be to create a `SessionDecorator` or something and have the decorator return the exact string. 

eg:

```
class ServiceProviderSessionDecorator
  def registration_heading
    t('headings.create_account_with_sp', sp: @sp_name)
  end
end
```

```
class SessionDecorator
  def registration_heading
     t('headings.create_account_without_sp')
  end
end
```

and then we'd return one decorator or the other depending on the presence on ivars.

Thoughts on this?